tests/regression/56-floats/01-base.c

```diff
@@ -0,0 +1,32 @@
+// PARAM: --enable ana.float.interval
+#include <assert.h>
+#include <float.h>
+#include <limits.h>
+#include <math.h>
+
+int main()
+{
+    double x, a = 2., b = 3. + 1;
+    float y, c = 2.f, d = 3.f + 1;
+
+    assert(x == 2.);  // UNKNOWN!
+    assert(y == 2.f); // UNKNOWN!
+
+    assert(a == 2.); // SUCCESS!
+    assert(a < 10.); // SUCCESS!
+    assert(a > 10.); // FAIL!
```

Comment:
`SUCCESS` doesn't actually do anything, by default assertions are required to succeed.
And we don't use `FAIL` with exclamation, only `UNKNOWN` (where it means actually unknowable, not unknown because of our imprecision).

Reply:
As we like the expressive `// SUCCESS` statements, we would like to keep them, but removed all `!` in places, where they didn't belong to. Also we tried to make a difference between `UNKNOWN` and `UNKNOWN!` now.